### ISSUE TITLE
jenkins-slave-ansible params should point to the rht-labs/labs-ci-cd repo

### ISSUE THREAD

**Issue by springdo** [rht-labs/labs-ci-cd#88]
The `SOURCE_REPOSITORY_URL` in the `params/jenkins-slave-ansible/build` file should reference the one in this repo; not @sherl0cks  one.... 
`SOURCE_REPOSITORY_URL=https://github.com/sherl0cks/labs-ci-cd`

Happy to PR and fix if it's helpful and not naggy!
:)

**Comment by sherl0cks** [comment]
yea I'm aware of this one. you are welcome to fix if you'd like.

I don't normally merge my own PR's, but the nature of committing a CI system sort of requires it. and of course skipping PR reviews means mistakes =)

**Comment by springdo** [comment]
Yeah man - I totes get it! Sometimes for progress / change you just gotta go for it ! I'll get PR to ya now shortly
